djstripe/settings.py

```diff
@@ -14,6 +14,7 @@
 
 INVOICE_FROM_EMAIL = getattr(settings, "DJSTRIPE_INVOICE_FROM_EMAIL", "billing@example.com")
 PAYMENTS_PLANS = getattr(settings, "DJSTRIPE_PLANS", {})
+HIERARCHY_PLANS = getattr(settings, "DJSTRIPE_HIERARCHY_PLANS", {})
```

Comment:
@kavdev I just copy the format from 
DJSTRIPE_INVOICE_FROM_EMAIL -> INVOICE_FROM_EMAIL
then:
DJSTRIPE_HIERARCHY_PLANS -> HIERARCHY_PLANS

But I can change it :) no problem.